src/api/session.tsx

```diff
@@ -0,0 +1,28 @@
+import client from './httpClient/index';
+
+interface User {
+  email: string;
+  password: string;
+  name?: string;
+}
+
+const login = (User: { email: string; password: string }): Promise<User> => {
+  return client.post('/auth/sign_in', JSON.stringify({ user: { ...User } }));
```

Comment:
WDYT if you replace all the hardcoded endpoints that you wrote (for example `/auth/sign_in`) and replace them with constants? Or Enums? 
It could look like this:
```
enum RouterPaths {
    HOME = '/',
    LOG_IN = '/login',
    SIGN_UP = '/sign-up',
    ...
}
```
It's going to make the code cleaner and more scalable.